tests/examples/opinions/united_states/tex_example.compare.json

```diff
@@ -1,142 +1,182 @@
 [
   {
-    "case_dates": "2014-07-03",
-    "case_names": "No case names fetched during tests.",
-    "download_urls": "tests/examples/opinions/united_states/RetrieveDocument.aspx?DocId=2457&Index=***sc%5cOpinion",
+    "case_dates": "2025-05-23",
+    "case_names": "Tenaris Bay City Inc. v. Ricky Ellisor",
+    "download_urls": "/media/1460582/230808.pdf",
     "precedential_statuses": "Published",
     "blocked_statuses": false,
     "date_filed_is_approximate": false,
-    "docket_numbers": "12-0617",
-    "case_name_shorts": ""
+    "dispositions": "The Court reverses the court of appeals' judgment and renders judgment.",
+    "docket_numbers": "23-0808",
+    "judges": "Chief Justice Blacklock",
+    "case_name_shorts": "",
+    "joined_by": "",
+    "per_curiam": false,
+    "types": "020lead"
   },
   {
-    "case_dates": "2014-07-03",
-    "case_names": "No case names fetched during tests.",
-    "download_urls": "tests/examples/opinions/united_states/RetrieveDocument.aspx?DocId=2456&Index=***sc%5cOpinion",
+    "case_dates": "2025-05-23",
+    "case_names": "Renaissance Medical Foundation v. Rebecca Lugo, Individually and as Next Friend of I.B.",
+    "download_urls": "/media/1460581/230607c.pdf",
     "precedential_statuses": "Published",
     "blocked_statuses": false,
     "date_filed_is_approximate": false,
-    "docket_numbers": "12-0804",
-    "case_name_shorts": ""
+    "dispositions": "The Court affirms the court of appeals' judgment and remands the case to the trial court.",
+    "docket_numbers": "23-0607",
+    "judges": "Justice Busby",
+    "case_name_shorts": "",
+    "joined_by": "Chief Justice Blacklock,Justice Devine",
```

Comment:
no spacing here... we probably should just drop this field if its hard to extract out